packages/vm/test/api/EIPs/eip-2935-historical-block-hashes.spec.ts

```diff
@@ -45,7 +45,7 @@ function eip2935ActiveAtCommon(timestamp: number, address: bigint) {
     Hardfork.GrayGlacier,
     Hardfork.Shanghai,
     Hardfork.Paris,
-    Hardfork.MergeForkIdTransition,
+    Hardfork.mergeNetsplitBlock,
```

Comment:
```suggestion
    Hardfork.MergeNetsplitBlock,
```

All HFs / enums are uppercase in our repo :)